api/notd/api/response_builder.py

```diff
@@ -58,6 +62,15 @@ async def collection_token_from_model(self, tokenMetadata: TokenMetadata) -> Api
             description=tokenMetadata.description,
             attributes=attributes,
         )
+    async def collection_token_from_registry_addresses_token_ids(self, tokens: Sequence[TokenTransfer]) -> List[ApiCollectionToken]:
```

Comment:
this function name and input/output is crazy. you take in transfesr and return tokens. this is definitely not the right structure. your manager should do complex stuff and the api should just serialize it. the manager should be returning tokens directly if thats what the api is gonna return